docs/docs/examples-tutorials/recipes/running-tracetest-with-dynatrace.md

```diff
@@ -0,0 +1,356 @@
+# Running Tracetest With Datadog
+
+:::note
+[Check out the source code on GitHub here.](https://github.com/kubeshop/tracetest/tree/main/examples/tracetest-datadog)
+:::
+
+[Tracetest](https://tracetest.io/) is a testing tool based on [OpenTelemetry](https://opentelemetry.io/) that allows you to test your distributed application. It allows you to use data from distributed traces generated by OpenTelemetry to validate and assert if your application has the desired behavior defined by your test definitions.
+
+[Datadog](https://www.datadoghq.com/) is an observability solution for cloud-scale applications, providing solutions to monitor databases, servers, tools and services. It provides integrated distributed tracing, logs and metrics solutions and supports OpenTelemetry standards.
```

Comment:
Here we can change to a specific description for Dynatrace, maybe something like this? (partially inspired by this [Wikipedia description](https://en.wikipedia.org/wiki/Dynatrace))

```suggestion
[Dynatrace](https://www.dynatrace.com/) is an observability solution used to monitor and optimize application performance, software development and security practices, IT infrastructure, and user experience. It provides integrated distributed tracing, logs, and metrics solutions and supports OpenTelemetry standards.
```